man/systemd.resource-control.xml

```diff
@@ -710,6 +716,42 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>BPFProgram=<replaceable>BPF_ATTACH_TYPE</replaceable><constant>:</constant><replaceable>BPF_FS_PROGRAM_PATH</replaceable></varname></term>
+        <listitem>
+          <para>Add a custom cgroup BPF program.</para>
+
+          <para><varname>BPFProgram=</varname> allows attaching cgroup-bpf hooks to a systemd unit including
+          egress and ingress attachment points also exposed to <varname>IP(Egress|Ingress)FilterPath=</varname>.
+          Cgroup-bpf hooks in the form of BPF programs loaded to the BPF filesystem are attached with cgroup-bpf attach
+          flags determined by the unit. For details about attachment types and flags see <ulink
+          url="https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/bpf.h"/></para>
```

Comment:
maybe better use git.kernel.org in the URI here, as the official kernel host:

https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/plain/include/uapi/linux/bpf.h